src/modules/contract/components/Contract/Contract.tsx

```diff
@@ -19,6 +19,7 @@ import './Contract.css';
 import { ContractFactory } from 'ethers';
 
 import networks from '../../../../../config/networks.json';
+import axios from 'axios';
```

Comment:
Maybe not useful to add another dependency? A fetch is enough for the usage we have